includes/admin/extensions/class-extensions-download-url.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace EDD\Admin\Extensions;
+
+class ExtensionsDownloadURL {
+
+	/**
+	 * Returns the download URL for an extension.
+	 *
+	 * @since 2.11.x
+	 *
+	 * @param int $id Extension ID.
+	 *
+	 * @return string
+	 */
+	public function get_url( $id, $license_key ) {
+		return $this->get_download_url( $id, $license_key );
+	}
+
+	/**
+	 * Fetch extension URLs from the remote server.
+	 *
+	 * @since 2.11.x
+	 *
+	 * @return bool|array False if no key or failure, array of extension URLs data otherwise.
+	 */
+	protected function get_download_url( $item_id = false, $license = false ) {
+		if ( ! $item_id || ! $license ) {
+			return false;
+		}
+
+		$option_name = "edd_extension_{$item_id}";
+		$option      = $this->get_stored_extension_data( $option_name );
+		if ( $option ) {
+			return $option['url'];
+		}
+
+		$request = wp_remote_post(
+			$this->get_api_url(),
+			array(
+				'timeout'   => 15,
+				'sslverify' => true,
+				'body'      => array(
+					'edd_action'  => 'get_version',
+					'license'     => $license,
+					'item_id'     => $item_id,
+					'php_version' => phpversion(),
+					'wp_version'  => get_bloginfo( 'version' ),
+				),
+			)
+		);
+
+		// If there was an API error, set transient for only 10 minutes.
+		if ( ! $request || $request instanceof WP_Error ) {
```

Comment:
```suggestion
		if ( ! $request || $request instanceof \WP_Error ) {
```

Because `WP_Error` isn't imported at the top of the file, it needs the backslash.